httpobsdashboard/conf/deviations.json

```diff
@@ -180,5 +180,42 @@
     "tlsobs": {
       "pass": true
     }
+  },
+  "HTTP Only": {
+    "httpobs": {
+      "tests": {
+        "strict-transport-security": {
```

Comment:
For readability, could you keep these in alphabetical order, so `redirection` and then `strict-transport-security`?